package.json

```diff
@@ -5,10 +5,10 @@
     "@types/node": "^13.1.7",
     "@types/normalize-url": "^1.9.1",
     "@types/ws": "0.0.39",
-    "@types/vscode": "1.41.0",
+    "@types/vscode": "1.43.0",
```

Comment:
should it be the same as [here](https://github.com/TypeFox/monaco-languageclient/blob/98f15c6499f23c853882c557dc18ccf7406ed1cf/client/package.json#L8)?